### DISCUSSION TITLE
I want fragmentation layouts

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#29] [Wants]
In other media, we have the ability to define content regions within a layout and flow content from one region into the next in a specific order. We don’t have this capability on the web currently. In order to achieve anything close to what’s possible in print layout tools, we either have to involve JavaScript or we must manually break our content into sections and hope we’ve allocated enough space for each of the blocks.

<hr>

https://webwewant.fyi/wants/5e5b546b5d7825f4f22d90ff

**Comment by aarongustafson** [comment]
This sounds like [CSS Regions](https://www.w3.org/TR/css-regions-1/). I believe that’s being picked back up in the CSS Working Group. Maybe @jensimmons or @rachelandrew can comment?

Current Status: 👎🏻 https://www.caniuse.com/#feat=css-regions

**Comment by aarongustafson** [comment]
Submitted for consideration at SFHTML5 (#30)

**Comment by rachelandrew** [comment]
yes that's Regions but there is no movement on it as far as I know.

My AEA talk last year was essentially about this (the last part anyway) https://aneventapart.com/news/post/making-things-better-aea-video

**Comment by aarongustafson** [comment]
@rachelandrew Thanks for that! I thought I remembered you bringing it up at AEA. I think Jen mentioned it on _Shop Talk Show_ last week too.

**Comment by jensimmons** [comment]
I would say this (my personal take, not an official statement from anyone but me): 
- The CSSWG understands the importance of solving this need, and many members want to make it happen. 
- The CSS Regions specification, as currently written, will not be implemented by browsers. It's sort of a solution to this need, but had too many problems to get consensus & agreement.
- There are other ideas being worked on. ”Fragmentation” is a good word to use when talking about it. Perhaps we'll end up with `overflow:fragment;` or something. 
- There's not a lot happening at the moment with writing a new spec, but there is a lot happening with the browser engines themselves. Engineers at several browsers are working on redoing or fixing-up their layout engine to make some kind of fragmented layout proposal possible. That's a needed next step, and it's underway right now with the plan of working on the rest later.

**Comment by caraya** [comment]
Thank you @jensimmons for the update!

**Comment by aarongustafson** [comment]
Thank you for chiming in with all of this detail @jensimmons!

**Comment by ghost** [comment]
@aarongustafson @jensimmons @caraya @rachelandrew Hi all! 
1. I recently opened this feature here "I want formatting a table in CSS like excel", [WebWeWant/webwewant.fyi\issue\555](https://github.com/WebWeWant/webwewant.fyi/issues/555) and here [Draft Idea/Spec/Proposal - Formatting a table in css like excel](https://discourse.wicg.io/t/draft-idea-spec-proposal-formatting-a-table-in-css-like-excel/5868)
2. My proposal is to format the html table as excel, example a1:a2{ ... } - it's a fragment layout. So... the resource I request is something similar here
